packages/frontend/workspace-impl/src/cloud/blob.ts

```diff
@@ -31,14 +32,26 @@ export const createAffineCloudBlobStorage = (
     },
     set: async (key, value) => {
       // set blob will check blob size & quota
-      const result = await fetcher({
+      return await fetcher({
         query: setBlobMutation,
         variables: {
           workspaceId,
           blob: new File([value], key),
         },
-      });
-      return result.setBlob;
+      })
+        .then(res => res.setBlob)
+        .catch(err => {
+          if (err[0].extensions && err[0].extensions.code === 413) {
```

Comment:
You still need to add a condition here `if (err instanceof GraphQLError)`

If `err` is not a `GraphQLError` here [0] may also raise other errors.